src/utils/__tests__/date.utils.test.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/jest-dom/extend-expect'
```

Comment:
Also instead of importing this in every file we should probably do something like setupFiles: ['@testing-library/jest-dom/extend-expect'] in jest.config.js

Reply:
Looks like we already have this in jest.config.js, do we need to import it at all then?...